wp-theme-2018/sidebar.php

```diff
@@ -54,15 +86,48 @@
         <aside class="nav-aside-wrapper">
             <nav id="nav-aside" class="nav-aside" role="navigation" aria-describedby="nav-aside-title">
                 <h2 class="h5 sr-only-xl"><?php esc_html_e("In the same section", 'epfl') ?></h2>
-                <?php
-                wp_nav_menu( array(
-                    'theme_location' => $EPFL_MENU_LOCATION,
-                    'menu_class'=> 'nav-menu',
-                    'container' => 'ul',
-                    'submenu' => get_the_ID(),
-                    'submenu_type' => $asideContent
-                ) );
-                ?>
+                <ul id="menu-main" class="nav-menu">
+                    <?php
+                        $items = get_all_menu_items_flattened();
+
+                        $current_item = get_current_item($items);
+
+                        // fullfil crumb_items array, in accordance with the items hierarchy
+                        $crumb_items = [];
+                        $crumb_item = $current_item;
+                        if (count($items) > 1) {
+                            while($crumb_item !== false)
+                            {
+                                array_unshift($crumb_items, $crumb_item);
+
+                                $index = (int) $crumb_item->menu_item_parent;
+                                $crumb_item = array_key_exists($index, $items)? $items[$index]: false;
+                            }
+                        } else {
+                            // make at least the only element printed
+                            $crumb_items = [$crumb_item];
+                        }
+
+                        $urls = get_current_url_and_homepage();
+                        $parent_items = get_stitched_menus($urls['home_page_url'], $urls['current_url'], $urls['current_lang']);
```

Comment:
Please either change `get_stitched_menus()` to accept a single object parameter; or make it an instance method. See other comment elsewhere.